test/components/views/messages/MPollBody-test.tsx

```diff
@@ -442,14 +460,14 @@ describe("MPollBody", () => {
         expect(body.html()).toBe("");
     });
 
-    it("renders nothing if poll has more than 20 answers", () => {
-        const answers = [...Array(21).keys()].map((i) => {
-            return { "id": `id${i}`, "org.matrix.msc1767.text": `Name ${i}` };
+    it("renders the first 20 answers if 21 were given", () => {
```

Comment:
From the spec: `answers must be an array with at least 1 option and no more than 20. Lengths outside this range are invalid and must not be rendered by clients.`

I understood this to mean that the poll is invalid in this case?

Reply:
The MSC was intending to say truncated, but clearly I didn't think to use that word 🤦‍♂️ 

"outside this range are invalid" means to say that values *within* the range are valid but anything outside of it is to be ignored.

Reply:
I'll update the MSC to clarify.